app/queries/servers/role.ts

```diff
@@ -3,12 +3,22 @@
 
 import {Database} from '@nozbe/watermelondb';
 
-import {MM_TABLES} from '@constants/database';
-import Role from '@typings/database/models/servers/role';
+import {Database as DatabaseConstants} from '@constants';
 
-const {SERVER: {ROLE}} = MM_TABLES;
+import type RoleModel from '@typings/database/models/servers/role';
+
+const {ROLE} = DatabaseConstants.MM_TABLES.SERVER;
 
 export const queryRoles = async (database: Database) => {
-    const roles = await database.collections.get(ROLE).query().fetch() as Role[];
+    const roles = await database.collections.get(ROLE).query().fetch() as RoleModel[];
```

Comment:
I like it better to use `get<RoleModel>(...` than `... as RoleModel[]`. Seems more "typescript like".

Reply:
I like that better, but haddened applied it to the get() function of a database operator.  I changed it in two locations.